assets/scripts/src/choices.js

```diff
@@ -1508,7 +1508,7 @@ class Choices {
     // Run callback if it is a function
     if (this.input === document.activeElement) {
       // Check that we have a value to search and the input was an alphanumeric character
-      if (value && value.length >= this.config.searchFloor) {
+      if (value.length >= this.config.searchFloor) {
```

Comment:
If value is ```null``` this will cause a JS error. Why did you remove the value check?

Reply:
You should take a look a little bit above: there is check for empty `value` already.